sql/server/dev/Dockerfile

```diff
@@ -14,7 +14,6 @@ FROM eclipse-temurin:20 as dev
 
 WORKDIR /skdb
 
-COPY --from=skdb /skdb/build ./build
```

Comment:
We need skdb/build for the skdb binary that the dev server uses. Without this we don't have the binary.